### PR TITLE
Feature fixed shrey

### PR DESCRIPTION
**Pull request by shrey**
### Description

Now it should work as i removed the try and catch block

##### Checklist

- [ ] `npm test` passes
- [ ] `npm run build` works
- [ ] tests are included
- [ ] documentation is changed or added

### CODE REVIEW

File: packages/offix-scheduler/src/store/OfflineStore.ts [hunk 1]
```diff
@@ -30,23 +26,21 @@ export class OfflineStore<T> {
...
       const serialized = this.serializer.serializeForStorage(entry);
       const offlineKey = this.getOfflineKey(entry.qid);
       // only add the offline key to the arrray if it's not already there
       if (!this.arrayOfKeys.includes(offlineKey)) {
         this.arrayOfKeys.push(offlineKey);
         await this.storage.setItem(this.offlineMetaKey, this.arrayOfKeys);
-      
       await this.storage.setItem(offlineKey, serialized);
-      debug("Saved Item");
+      Debug("serialized:item");
```
Comment (wtrocki) [inline packages/offix-scheduler/src/store/OfflineStore.ts:43]: Debug is pointless if we do not know **what** we serialized.

File: packages/offix-scheduler/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+module.exports = {
```
Comment (wtrocki) [inline packages/offix-scheduler/.eslintrc.js:1]: Let's avoid adding changes not relevant to PR

Comment (shrey) [reply packages/offix-scheduler/.eslintrc.js:1]: Okay

**Comment by kingsleyzissou** [comment]
@shrey see the comments I made on the other pull request that was made

**Pull request closed without merging**